test/minikube/minikube_external_access_test.go

```diff
@@ -74,14 +74,31 @@ func getNuoSQLVersion(t *testing.T) *semver.Version {
 		require.NoError(t, err)
 	}
 	match := regexp.MustCompile("NuoDB Client build (.*)").FindStringSubmatch(string(out))
-	require.NotNil(t, match, out)
-	// strip comment from semantic version
-	versionStr := strings.Split(match[1], "-")[0]
+	require.NotNil(t, match, string(out))
+	versionStr := normalizeNuoSQLVersion(match[1])
 	version, err := semver.NewVersion(versionStr)
-	require.NoError(t, err)
+	require.NoError(t, err, version)
```

Comment:
The `version` variable will probably be `nil`. The `versionStr` should be printed instead.